SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -187,9 +189,9 @@ + (YapDatabaseDeserializer)logOnFailureDeserializer
             OWSProdErrorWParams(@"storage_error_deserialization", ^{
                 return (@{
```

Comment:
It seems like we're trying to add one extra parameter to the standard AnalyticsParametersFromNSException. You don't have to do it now, but it would be nice to be able to specify "additionalParams" which would mix with the default params.